meta-oe/recipes-connectivity/hiba/hiba_git.bb

```diff
@@ -0,0 +1,52 @@
+SUMMARY = "Google Host Identity Based Authorization"
+DESCRIPTION = "HIBA is a system built on top of regular OpenSSH \
+  certificate-based authentication that allows to manage flexible \
+  authorization of principals on pools of target hosts without the need to \
+  push customized authorized_users files periodically."
+
+S = "${WORKDIR}/git"
+
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=7e161abe9a4585310725dd86c28d1ae2"
+
+SRC_URI = "git://github.com/google/hiba;protocol=https;branch=main"
+SRCREV = "98b551e2a0297b1872851bb81b6d3a2b682778ee"
+
+# HIBA Internal OpenSSH Version (Should match the version in meta-oe).
```

Comment:
This is actually not an issue: HIBA only uses a small part of the libssh which happens to be part of the RFC and thus pretty stable, so we can technically use pretty much any OpenSSH version.

Suggestion:

Internal OpenSSH version for HIBA to link against (the version doesn't have to match the actual installed version).